src/services/mongoose/phone-code.ts

```diff
@@ -0,0 +1,54 @@
+import { VALIDITY_TIME_CODE } from "@config/app"
+import {
+  CouldNotFindPhoneCodeError,
+  RepositoryError,
+  UnknownRepositoryError,
+} from "@domain/errors"
+import moment from "moment"
+import { PhoneCode } from "./schema"
+
+export const PhoneCodesRepository = (): IPhoneCodesRepository => {
+  const findRecent = async ({
```

Comment:
The definition of `recent` is business logic and should be hard coded in the repo - it should be part of the domain.

perhaps something like:
```
existsNewerThan({phone: PhoneNumber, code: PhoneCode, age: Seconds})
```